examples/get_unchecked_address.rs

```diff
@@ -0,0 +1,48 @@
+//! Get an unchecked from seed. This is just for developing purpose in case you want to see certain address.
```

Comment:
```suggestion
//! Get an address from seed and index that the address could be an used address on the Tangle. This is just for developing purpose in case you want to see a certain address.
```